docs/source/docker/advanced/datapool.rst

```diff
@@ -3,7 +3,7 @@
 Datapool
```

Comment:
Side question: Everything in `docs/source/docker` is hidden, right? At least I can't find it in the SSL docs.

Reply:
I just searched for "lightly" and changed everything as the structure of the docs are a bit weird. Very possible that I adjusted things that were not visible (not sure why we even still have them?)

Reply:
I don't know :)